cloud/azure/main.tf

```
@@ -75,127 +55,60 @@ locals {
   redirect_configuration_name    = "${azurerm_virtual_network.civiform_vnet.name}-rdrcfg"
 }
 
-# resource "azurerm_application_gateway" "network" {
-#   name                = "civiform-appgateway"
-#   resource_group_name = azurerm_resource_group.rg.name
-#   location            = azurerm_resource_group.rg.location
-
-#   sku {
-#     name     = "Standard_Small"
-#     tier     = "Standard"
-#     capacity = 2
-#   }
-
-#   gateway_ip_configuration {
-#     name      = "civiform-gateway-ip-configuration"
-#     subnet_id = azurerm_subnet.server_subnet.id
-#   }
-
-#   frontend_port {
-#     name = local.frontend_port_name
-#     port = 80
-#   }
-
-#   frontend_ip_configuration {
-#     name                 = local.frontend_ip_configuration_name
-#     public_ip_address_id = azurerm_public_ip.server_container_group_ip.id
-#   }
-
-#   backend_address_pool {
-#     name = local.backend_address_pool_name
-#   }
-
-#   backend_http_settings {
-#     name                  = local.http_setting_name
-#     cookie_based_affinity = "Disabled"
-#     path                  = "/path1/"
-#     port                  = 80
-#     protocol              = "Http"
-#     request_timeout       = 60
-#   }
-
-#   http_listener {
-#     name                           = local.listener_name
-#     frontend_ip_configuration_name = local.frontend_ip_configuration_name
-#     frontend_port_name             = local.frontend_port_name
-#     protocol                       = "Http"
-#   }
-
-#   request_routing_rule {
-#     name                       = local.request_routing_rule_name
-#     rule_type                  = "Basic"
-#     http_listener_name         = local.listener_name
-#     backend_address_pool_name  = local.backend_address_pool_name
-#     backend_http_settings_name = local.http_setting_name
-#   }
-# }
-
-resource "azurerm_network_profile" "civiform_network_profile" {
-  name                = "civiform-network-profile"
+resource "azurerm_app_service_plan" "plan" {
+  name                = "${azurerm_resource_group.rg.name}-plan"
   location            = azurerm_resource_group.rg.location
   resource_group_name = azurerm_resource_group.rg.name
 
-  container_network_interface {
-    name = "civiform-server-network-interface"
+  # Define Linux as Host OS
+  kind     = "Linux"
+  reserved = true # Mandatory for Linux plans
 
-    ip_configuration {
-      name      = "civiform-network-interface-ip-configuration"
-      subnet_id = azurerm_subnet.server_subnet.id
-    }
+  # Choose size
+  sku {
+    tier     = "Standard"
+    size     = "S1"
+    capacity = "2"
   }
 }
 
-resource "azurerm_container_group" "cg" {
-  name                = "civiform-container-group"
+resource "azurerm_app_service" "civiform-app" {
```

suggestion (blocking): let's standardize on underscores unless we'd like to standardize on something else

```suggestion
resource "azurerm_app_service" "civiform_app" {
```

---

done